.travis.yml

```diff
@@ -1,6 +1,22 @@
-language: node_js
-node_js:
-  - "0.10"
-  - "0.11"
-  - "0.12"
-  - "4.2.4"
+language: c++
+env:
+  - NODE_VERSION="0.10"
+  - NODE_VERSION="0.11"
+  - NODE_VERSION="0.12"
+  - NODE_VERSION="4.2.4"
```

Comment:
@ariporad Finally got around to fixing this. You're right, 4.x.x is a better move going forward. This should be good for review now.